README.md

```diff
@@ -113,6 +113,32 @@ app = Litestar(route_handlers=[hello_world])
 for testing and POC work.
 </details>
 
+## Sponsors
+
+Litestar is an open-source project, and we enjoy the support of our sponsors to help fund the exciting
+work we do.
+
+If you would like to support the work that we do please consider [becoming a sponsor][sponsor-github]
+on [GitHub][sponsor-github] or [Open Collective][sponsor-oc].
+
+We also participate in pledge-based sponsorship with [Polar][sponsor-polar].
+
+
+[//]: # "Note to maintainers: Highest sponsors first; no more than 3 per row - create new div if needed"
+
+<div style="display: flex; justify-content: center; align-items: center; gap: 8px;">
+  <a href="https://scalar.com">
+    <img src="https://raw.githubusercontent.com/litestar-org/branding/main/assets/sponsors/scalar.svg" alt="Scalar.com" style="border-radius: 10px; width: auto; max-height: 150px;"/> <!-- Adjust max-height as needed -->
+  </a>
+  <a href="https://telemetrysports.com/">
+    <img src="https://raw.githubusercontent.com/litestar-org/branding/main/assets/sponsors/telemetry-sports-temp.svg" alt="Telemetry Sports" style="border-radius: 10px; width: auto; max-height: 150px;"/> <!-- Adjust max-height as needed -->
+  </a>
+</div>
```

Comment:
Can't we autogenerate this section? I can see this getting outdated fairly quickly otherwise.

Reply:
Hopefully we can do this via #2866 soon.